src/compas/geometry/polyhedron.py

```diff
@@ -9,6 +9,7 @@
 from compas.geometry import Polygon
 from compas.geometry import Point
 from compas.geometry import Line
+from compas.tolerance import TOL
 from .geometry import Geometry
```

Comment:
since you're very close, could you make this import absolute instead? (unless there's any reason for the relative one, but I don't see any